raiden-cli/src/routes/api.v1.ts

```diff
@@ -84,5 +84,16 @@ export function makeApiV1Router(this: Cli): Router {
     response.json(RaidenConfig.encode(this.raiden.config));
   });
 
+  router.get('/state.json', async (_request: Request, response: Response) => {
+    response.writeHead(200, { 'Content-Type': 'application/json' });
+    response.write('[');
+    let count = 0;
+    for await (const row of this.raiden.dumpDatabase()) {
+      response.write((count++ ? ',\n' : '\n') + JSON.stringify(row));
```

Comment:
It looks like we're trying to do manually what `JSON.stringify()` would do for us. Can't we just load the result of `dumpDataBase()` in a variable (assuming it returns a list) and then just use `stringify` once?

Reply:
Using `stringify` on the whole thing will store everything in memory. The idea here is exactly to _stream_ the contents (which can grow very big) one row at a time, iterating through the `AsyncIterable` and writing each item. This is specially useful here in cli, as its database is more probable to grow indefinitely.